src/features/WordTable/TableHeader/index.tsx

```diff
@@ -49,7 +60,9 @@ export const TableHeader = ({
             </Button>
 
             <Button
-              onMouseDown={() => saveFlashcards(folderName, cards)}
+              onMouseDown={(event) => event.shiftKey || event.metaKey 
+                ? copyFlashcards(folderName, cards.reverse()) 
```

Comment:
This feels like a hacky fix... `filteredCards` already has the cards reversed. Maybe that can be passed as a prop instead.

Reply:
Lol I put like 0 thought into this problem yeah that's valid